test/test_cpu.cpp

```diff
@@ -334,6 +350,20 @@ void test_cpu(std::vector<std::string>& user_input_options) {
                 write_dates(test_dates, test_num, date);
                 test_dates[test_num] = date;
             }
+
+            if (reinizialize_test_outputs == "all" ||
+                reinizialize_test_outputs == "lstm") {
+                // Reinizialize test outputs for the series forecasting task
+                std::cout << "Reinizializing LSTM test outputs" << std::endl;
+
+                test_lstm_cpu(true, date, "lstm", "time_series");
+
+                test_num = 4;  // LSTM
+
+                // Update de last date of the test
+                write_dates(test_dates, test_num, date);
+                test_dates[test_num] = date;
+            }
```

Comment:
Could we have a summary for unittest? e.g., for a total of 9 test, how many test has failed or passed at the end

Reply:
I've tried to create a failed test. I manually modify one of the values of `2023_03_18_forward_hidden_states_fnn_1D.csv`. Here is its outputs:
```shell
Performing all tests
v1[0][31] = -0.508126
v2[0][31] = -0.708126
Test for FNN FORWARD HIDDEN STATES has FAILED in 1D data

[ 11%] FNN tests failed
[ 22%] FNN heteroscedastic tests passed
[ 33%] FNN full covariance tests passed
[ 44%] FNN derivatives tests passed
[ 55%] LSTM tests passed
```
Could we specify what are v1 and v2? For this case,  v1 and v2 are the `forward_hidden_state_fnn_1D`

Reply:
If the output would be like this: 

`Different values in fnn bias for Boston_housing data
ref_vector[0][5] = -0.04223
test_vector[0][5] = -0.0459223
Test for FNN PARAMS has FAILED in Boston_housing data

[ 11%] FNN tests failed
[ 22%] FNN heteroscedastic tests passed
[ 33%] FNN full covariance tests passed
[ 44%] FNN derivatives tests passed
[ 55%] LSTM tests passed
`
Would you leave then the "Test for FNN PARAMS has FAILED in Boston_housing data" error message?

Reply:
Yep that would be a good idea. So we have three error message
- Params
- Forward hidden states
- Backward hidden states